.github/workflows/push-gradle-ci.yml

```diff
@@ -39,6 +39,36 @@ jobs:
     - name: Build with Gradle Wrapper
       run: ./gradlew release
 
+    - name: Upload release
+      if: github.repository == 'ion-fusion/fusion-java'
```

Comment:
[non-blocking] Would be nice if we could provide this flag as external configuration on the repository, to enable testing in a fork.

Reply:
Good idea! Looks like we can set repo-level variables to use in place of this. I'll submit a separate PR changing that; it will dovetail nicely with other changes I want to make.